### ISSUE TITLE
Need to mention level AA in EAA mapping document? 

### ISSUE THREAD

**Issue by plannero** [mattgarrish/epub-revision#2583]
The second note under [section 3.1.1 in Epub Accessibility 1.1](https://www.w3.org/TR/epub-a11y-11/#sec-wcag-conf) makes it clear that "EPUB publications will need to meet more than just the basic Level A success criteria to be compliant with these laws." (referring to EAA, for example). 

However, in the EAA-EPUB mapping document there is only [one mention](https://www.w3.org/TR/epub-a11y-eaa-mapping/#annex-I-section-III-b-v) of (two) AA level SC that are required by EAA. Should I conclude from this that the EAA requires no other AA level SC:s for e-books? 

There are several references to the entire WCAG 2 in the mapping document. Perhaps these references should be changed to "WCAG 2 level AA", since EU accessibility regulation generally refers to EN301549, which usually targets AA level.

Or is such a clarification not the role of this mapping document but rather something that must be indicated in an implementing decision from the EU commission?
